core/server/master/src/main/java/alluxio/master/meta/MetaMaster.java

```diff
@@ -203,4 +204,10 @@ void setPathConfiguration(String path, Map<PropertyKey, String> properties)
    * @return a list of status
    */
   List<ProxyStatus> listProxyStatus();
+
+  /**
+   * @param version a version to filter the updated configurations
+   * @return the updated properties after the given version
+   */
+  Pair<List<Map<String, String>>, Long> getUpdatedConfiguration(long version);
```

Comment:
let's define an obj for this complicated data structure

Reply:
done